### ISSUE TITLE
Eetherscan: Unable to locate the Transaction entry

### ISSUE THREAD

**Issue by BlatzCoin** [MetaMask/metamask-extension#1906]
**Briefly describe the issue you've encountered
  *  Expected Behavior**
Transaction to process

  ***  Actual Behavior**

Unfortunately I wan't paying close attention so I currently have **11 transactions** that

Get error in Etherscan  Unable to locate the Transaction entry

https://etherscan.io/tx/0xf0b034e72d6b82ae797097b5bac993e7427b85055224873f3ff738f75b8be73a

![screenshot 2017-08-12 13 12 38](https://user-images.githubusercontent.com/27861868/29243286-d356ba20-7f60-11e7-8fa3-5300fd64ef51.png)

... indicates pending  and link indicates loading

https://metamask.github.io/eth-tx-viz/?tx=0xf0b034e72d6b82ae797097b5bac993e7427b85055224873f3ff738f75b8be73a

Account balance was low but not empty so I successfully transferred 1-ETH to the wallet but I'm concerned to attempt to send it out since all outbound transactions seem stuck

This is an imported Mist wallet address since my Mist node is out of sync and unusable at this point. I tried to Lock Metemask and recover account via seed and re-importing mist wallet with no luck

  ***  Browser Used**
Chrome Version 60.0.3112.90 (Official Build) (64-bit)

Metamask 3.9.6

  ***  Operating System Used**

Win10 Pro 10.0.15063 x64

  Screenshots are very helpful and will expedite your issue being resolved!
-->

**Comment by BlatzCoin** [comment]
It took 24hrs but some of the transactions finally failed. I'm not sure if its related but I logged into my wallet from a different computer and the remainder completed. I suspect the issue came from there being two transaction #162. I'm now having an issue where my wallet balance doesn't reflect the transaction value. two .01 ether transaction reduced my balance .1 ether?

**Comment by BlatzCoin** [comment]
It took 24 hours but several transactions failed. I then logged into my wallet from another computer and that may have caused the remainder to complete. I think the issue may have been caused by the wallet having two transactions #162. I can now do transactions from the wallet on the new computer but I can transact from the old wallet without failing immediately. I'm going to create a new metamask wallet and import the mist wallet into that and see if that works.
